test/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -60,7 +84,7 @@ public TestAsset WithSource()
                         .Descendants(XName.Get("PackageReference", "http://schemas.microsoft.com/developer/msbuild/2003"))
                         .FirstOrDefault(pr => pr.Attribute("Include")?.Value == "Microsoft.NET.Sdk")
                         ?.Element(XName.Get("Version", "http://schemas.microsoft.com/developer/msbuild/2003"))
-                        ?.SetValue('[' +_buildVersion + ']');
+                        ?.SetValue('[' +BuildVersion + ']');
```

Comment:
nit: whitespace missing between + and BuildVersion. Also, consider $"[{BuildVersion}]".